scss/components/table.scss

```diff
@@ -0,0 +1,60 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-table
+    thead
+    tbody
+        tr+([aria-selected])
+*/
+$block: ns(table);
+
+.#{$block} {
+    //LOCAL VARS (set all themeable properties, always include !default)
+    $tn-table-border-color: tn-color(neutral, 3) !default;
+    $tn-table-cell-spacing: $tn-width--gutter !default;
+    $tn-table-cell-padding: tn-space(3) !default;
+
+    //BLOCK BASE *******************************************
+    //set all BLOCK reset and baseline styles
+    @include reset;
+    width: 100%;
+    max-width: 100%;
+    border-collapse: collapse;
+    tr {
+        &:hover {
+            background-color: lighten(tn-color(neutral, 3), 10);
+        }
+        &.is-selected,
+        &[aria-selected="true"] {
+            background-color: lighten(tn-color(neutral, 3), 5);
```

Comment:
@LeoT7508 The SASS lighten function did not return the exact colors you specified, so I matched as close as I could. I like using the same color - `neutral-3` - for hover and selected instead of different color, these shades come pretty close to what you have in Zeplin.